### ISSUE TITLE
can't disable multiple rules with disable-enable-pair.allowWholeFile

### ISSUE THREAD

**Issue by teppeis** [mysticatea/eslint-plugin-eslint-comments#9]
source
```js
/*eslint-disable no-unused-var, no-undef */
var foo = bar()
```

output
```
1:33  error  Requires 'eslint-enable' directive for 'no-undef'  eslint-comments/disable-enable-pair
```

repro
https://github.com/teppeis-sandbox/eslint-comments-repro

**Comment by mysticatea** [comment]
Thank you for the report.

I confirmed it. I will fix.

**Comment by teppeis** [comment]
@mysticatea thanks! I've confirmed it's fixed.
